homepairs/homepairsUI/tests/redux/property-list/UpdatePropertyTest.test.tsx

```diff
@@ -0,0 +1,133 @@
+import { PropertyListActions } from 'homepairs-redux-actions';
+import {
+    Property,
+    EditPropertyState,
+    UpdatePropertyAction,
+} from 'homepairs-types';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import { propertyManagerMock1 } from '../../fixtures/StoreFixture';
+
+const URL = 'https://homepairs-alpha.herokuapp.com/API/property/update/';
+
+const { postUpdatedProperty, PROPERTY_LIST_ACTION_TYPES } = PropertyListActions;
+
+const prevProperty: Property = {
+    streetAddress: 'Cool Aid Road',
+    city: 'HoodRat Town',
+    state: 'Georgia',
+    tenants: 4,
+    bedrooms: 3,
+    bathrooms: 3,
+};
+const updatedPropertyTest: Property = {
+    streetAddress: 'Cool Aid Drive',
+    city: 'HoodRat City',
+    state: 'Georgia',
+    tenants: 3,
+    bedrooms: 3,
+    bathrooms: 3,
+};
+const updatedPropertyCredentials: EditPropertyState = {
+    email: 'jackylynne@gmail.com',
+    index: 0,
+    oldProp: prevProperty,
+    roopairsToken: '1B88484KDKSLIEK058UDDD',
+};
+
+// To test nested actions with an async REST request, we need to call an await on the
+// dispatch methods in order to get an updated state.
+describe('postUpdatedProperty Action', () => {
```

Comment:
I would love to see more success tests that show some fields changing  vs all fields changing vs no fields changing

Reply:
We ended up discussing this and determined that separate allocation testing will occur for components and perhaps separate allocation methods for http requests